txn_writer.go

```diff
@@ -19,6 +19,18 @@ import (
 	"github.com/sdboyer/gps"
 )
 
+// Example string to be written to the manifest file
+// if no dependencies are found in the project
+// during `dep init`
+const exampleToml = `
+# Example:
+# [[dependencies]]
+# branch = "master"
+# name = "github.com/vendor/package"
+# revision = "abc123"
```

Comment:
Looks like we still need an example for source, `# source = "https://github.com/myfork/package.git"` will do.